pkg/util/platforms_test.go

```diff
@@ -0,0 +1,23 @@
+package util_test
+
+import (
+	"fmt"
+	"testing"
+
+	remoteexecution "github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2"
+	re_util "github.com/buildbarn/bb-remote-execution/pkg/util"
+)
+
+func TestSortPlatformProperties(t *testing.T) {
+	reverseSortedPlatform := &remoteexecution.Platform{
+		Properties: []*remoteexecution.Platform_Property{
+			{Name: "os", Value: "armv6"},
+			{Name: "cpu", Value: "linux"},
+		},
+	}
+
+	sortedPlatform := re_util.SortPlatformProperties(reverseSortedPlatform)
+	if sortedPlatform.Properties[0].Name == "os" {
```

Comment:
Please use the `require` package to make this consistent with the rest of Buildbarn's unit tests.

Also, be sure to compare the entire resulting list. You should be able to write this:
```go
require.Equal(
   t,
   []*remoteexecution.Platform_Property{
          ...,
   },
   re_util.SortPlatformProperties([]*remoteexecution.Platform_Property{
          ...,
   }))
```